src/_capi.pxd

```diff
@@ -451,27 +457,50 @@ cdef extern from "blend2d.h":
+#     BLResult blFontGetMatrix(const BLFontCore* self, BLFontMatrix* out)
+#     BLResult blFontGetMetrics(const BLFontCore* self, BLFontMetrics* out)
+#     BLResult blFontGetDesignMetrics(const BLFontCore* self, BLFontDesignMetrics* out)
+#     BLResult blFontGetTextMetrics(const BLFontCore* self, BLGlyphBufferCore* gb, BLTextMetrics* out)
+#     BLResult blFontGetGlyphBounds(const BLFontCore* self, const uint32_t* glyphData, intptr_t glyphAdvance, BLBoxI* out, size_t count)
+#     BLResult blFontGetGlyphAdvances(const BLFontCore* self, const uint32_t* glyphData, intptr_t glyphAdvance, BLGlyphPlacement* out, size_t count)
+#     BLResult blFontGetGlyphOutlines(const BLFontCore* self, uint32_t glyphId, const BLMatrix2D* userMatrix, BLPathCore* out, BLPathSinkFunc sink, void* closure)
+#     BLResult blFontGetGlyphRunOutlines(const BLFontCore* self, const BLGlyphRun* glyphRun, const BLMatrix2D* userMatrix, BLPathCore* out, BLPathSinkFunc sink, void* closure)
+
+    BLResult blFontInit(BLFontCore* self) ;
+    BLResult blFontDestroy(BLFontCore* self) ;
+    BLResult blFontReset(BLFontCore* self) ;
+    BLResult blFontAssignMove(BLFontCore* self, BLFontCore* other) ;
+    BLResult blFontAssignWeak(BLFontCore* self, const BLFontCore* other) ;
+    bool blFontEquals(const BLFontCore* a, const BLFontCore* b) ;
+    BLResult blFontCreateFromFace(BLFontCore* self, const BLFontFaceCore* face, float size) ;
+    BLResult blFontShape(const BLFontCore* self, BLGlyphBufferCore* gb) ;
+    BLResult blFontMapTextToGlyphs(const BLFontCore* self, BLGlyphBufferCore* gb, BLGlyphMappingState* stateOut) ;
+    BLResult blFontPositionGlyphs(const BLFontCore* self, BLGlyphBufferCore* gb) ;
+    BLResult blFontApplyKerning(const BLFontCore* self, BLGlyphBufferCore* gb) ;
+    BLResult blFontApplyGSub(const BLFontCore* self, BLGlyphBufferCore* gb, const BLBitArrayCore* lookups) ;
+    BLResult blFontApplyGPos(const BLFontCore* self, BLGlyphBufferCore* gb, const BLBitArrayCore* lookups) ;
+    BLResult blFontGetMetrics(const BLFontCore* self, BLFontMetrics* out) ;
+    BLResult blFontGetMatrix(const BLFontCore* self, BLFontMatrix* out) ;
+    BLResult blFontGetDesignMetrics(const BLFontCore* self, BLFontDesignMetrics* out) ;
+    BLResult blFontGetTextMetrics(const BLFontCore* self, BLGlyphBufferCore* gb, BLTextMetrics* out) ;
+    BLResult blFontGetGlyphBounds(const BLFontCore* self, const uint32_t* glyphData, intptr_t glyphAdvance, BLBoxI* out, size_t count) ;
+    BLResult blFontGetGlyphAdvances(const BLFontCore* self, const uint32_t* glyphData, intptr_t glyphAdvance, BLGlyphPlacement* out, size_t count) ;
+    BLResult blFontGetGlyphOutlines(const BLFontCore* self, BLGlyphId glyphId, const BLMatrix2D* userTransform, BLPathCore* out, BLPathSinkFunc sink, void* userData) ;
+    BLResult blFontGetGlyphRunOutlines(const BLFontCore* self, const BLGlyphRun* glyphRun, const BLMatrix2D* userTransform, BLPathCore* out, BLPathSinkFunc sink, void* userData) ;
+
```

Comment:
I'd prefer for the commented code here to be replaced by the new version introduced here. Minus the semicolons.

Reply:
I ll do that